PsfLauncher/main.cpp

```diff
@@ -13,223 +13,227 @@
 #include <psf_runtime.h>
 #include <shellapi.h>
 #include <combaseapi.h>
+#include <ppltasks.h>
+#include <ShObjIdl.h>
+#include "ErrorInformation.h"
 
-using namespace std::literals;
+//These two macros don't exist in RS1.  Define them here to prevent build

```

Comment:
>RS1 [](start = 34, length = 3)

but common.props is RS2 ...

Reply:
I'm confused by this comment.  What does common.props have to do with anything?

Reply:
that's where you're setting RS2 (I suggest you move it back to RS1)